go.mod

```
@@ -0,0 +1,9 @@
+module github.com/caylent/tamr-terraform-aws-s3
```

Should this be updated to point at the Tamr repo?
```suggestion
module github.com/Datatamer/tamr-terraform-aws-s3
```